linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -535,11 +545,12 @@ private[emitter] object CoreJSLib {
         val instance = varRef("instance")
         val v = varRef("v")
         defineFunction(name, paramList(instance), {
-          Switch(typeof(instance), List(
-              str("string") -> {
+          Switch(typeof(instance),
+              List(
+                  str("string") -> {
                 Return(constantClassResult(BoxedStringClass))
               },
```

Comment:
This also seems incorrect: The closing brace does not even have the same indentation level than the line it opens on.